automation/run-tests.sh

```diff
@@ -14,6 +14,7 @@ TEST_TYPE_UNIT_PY36="unit_py36"
 TEST_TYPE_UNIT_PY37="unit_py37"
 TEST_TYPE_UNIT_PY38="unit_py38"
 TEST_TYPE_INTEG="integ"
+TEST_TYPE_INTEG_NM="integ_nm"
```

Comment:
IMHO it would be nicer to have one target to run all tests locally like `integ` and introduce two new targets that will be called from `integ`.

Reply:
Fine by me.